src/style.css

```
@@ -1,3 +1,87 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  --color-link-hover: #00acdc;
+}
```

Create a variable with root selector
```suggestion
:root {
  --main-color: #00acdc;
}
```